fedmsg_meta_fedora_infrastructure/tests/mdapi.py

```diff
@@ -0,0 +1,407 @@
+# This file is part of fedmsg.
+# Copyright (C) 2015 Red Hat, Inc.
+#
+# fedmsg is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# fedmsg is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with fedmsg; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+#
+# Authors:  Pierre-Yves Chibon <pingou@pingoured.fr>
+#
+""" Tests for mdapi messages """
+
+import unittest
+
+from fedmsg_meta_fedora_infrastructure.tests.base import Base
+
+from .common import add_doc
+
+
+class TestNewProject(Base):
+    """ These messages are published when a new project is created on
+    `pagure <https://pagure.io>`_.
+    """
+    expected_title = "mdapi.repo.update"
+    expected_subti = 'mdapi meta-data update: files: +5/-4, '\
+        'enhances: +0/-0, recommends: +2/-1, supplements: +0/-0, '\
+        'obsoletes: +2/-2, provides: +2/-2, suggests: +2/-2, '\
+        'conflicts: +1/-2, packages: +2/-3, requires: +2/-2'
+    expected_link = "https://apps.fedoraproject.org/mdapi"
+    expected_icon = "https://apps.fedoraproject.org/packages/" + \
+        "images/icons/package_128x128.png"
+    expected_secondary_icon = ""
+    expected_packages = set([
+        'java-1.8.0-openjdk-devel-debug',
+        'vips',
+        'python3-celery',
+        'python3-dropbox',
+        'libfdisk',
+        'journal-brief',
+        'meataxe',
+        'rpm',
+        'akonadi-mysql',
+        'java-1.8.0-openjdk-headless',
+        'python-wrapt-doc',
+        'python3-yui(x86-64)',
+        'python-faker-doc',
+        'python3-PyQt4-devel',
+        'boost(x86-32)',
+        'PyOpenGL',
+        'audit-libs-devel',
+        'python-simplegeneric',
+        'python-aniso8601',
+        'python3',
+        'django-markdown2',
+        'python-pytools',
+        'tyrus-container-servlet',
+        'python3-devel',
+        'waf-python3',
+        'avahi-ui-sharp-devel(x86-64)',
+        'python2-XStatic-roboto-fontface',
+        'python-Traits(x86-64)',
```

Comment:
Can we add some code to strip these ``(x86-64)`` strings from the end?  They'll cause pattern matching problems in other projects.